idseq_dag/util/server.py

```diff
@@ -176,3 +180,21 @@ def wait_for_server_ip(service_name,
                                          max_concurrent, chunk_id)
         return result
 
+
+def get_instance_iD_from_iP(instance_iP):
+    cmd = f"aws ec2 describe-instances --filter Name=private-ip-address,Values={instance_iP} --query 'Reservations[].Instances[].[InstanceId]' --output=text"
+    results = command.execute_with_output(cmd).splitlines()
+    assert len(results) == 1
+    return results[0]
+
+
+def register_job_tag(instance_iP):
+    batch_job_id = os.environ.get('AWS_BATCH_JOB_ID', 'local')
+    job_tag = f"{JOB_TAG_PREFIX}{batch_job_id}"
```

Comment:
Is this sufficient?   Not one per chunk but one per job?   Even if multiple chunks from the same job are running on the same gsnap server?   But then when you delete the tag you are signaling all chunks are done, which is information you don't have at that time.